aiounifi/models/device.py

```diff
@@ -917,6 +961,35 @@ def wlan_overrides(self) -> list[TypedDeviceWlanOverrides]:
         """Wlan configuration override."""
         return self.raw.get("wlan_overrides", [])
 
+    @property
+    def hw_caps(self) -> int:
+        """Hardware capabilities."""
+        return self.raw.get("hw_caps", 0)
+
+    @property
+    def supports_led_ring(self) -> bool:
+        """Check if the hardware supports an LED ring based on the second bit of `hw_caps`."""
+        return bool(self.hw_caps & HardwareCapability.LED_RING)
+
+    @property
+    def led_override(self) -> str | None:
+        """LED override."""
+        return self.raw.get("led_override")
+
+    @property
+    def led_override_color(self) -> str | None:
+        """LED override color - if supports LED ring."""
+        if self.supports_led_ring:
+            return self.raw.get("led_override_color")
+
+    @property
+    def led_override_color_brightness(self) -> int | None:
+        """LED override color brightness - if supports LED ring."""
+        if self.supports_led_ring:
+            value = self.raw.get("led_override_color_brightness")
+            assert isinstance(value, int)
```

Comment:
```suggestion
            value = self.raw.get("led_override_color_brightness")
```
I saw a typo (`led_override_color_brightnes: int`) in `class TypedDevice(TypedDict):` if you add another s in brightness you shouldn't need to use the assert